### ISSUE TITLE
Accessibility : cannot focus radio/checkboxOne while no modality selected

### ISSUE THREAD

**Issue by QRuhier** [InseeFr/Lunatic#1036]
## Describe the bug

On radio/checkboxOne, if there is no modality selected, you cannot focus on the component with keyboard navigation

- [x] I tested it on the storybook, which leads me to believe that it's a lunatic bug.

## Browser where the bug was tested

- [ ] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge
- [ ] Other (please specify): ******\_\_\_******

**Comment by QRuhier** [comment]
no subject for Stromae-v3 considering that it is handled on lunatic-dsfr
